starter-code/public/styles/style.css

```diff
@@ -0,0 +1,128 @@
+.background {
+  height: 100vh;
+  width: 100vw;
+  background-image: url("/images/spotify-background.jpeg");
+  background-size: 100%;
```

Comment:
This may look a bit better if you're making the webpage responsive? Background-size: 100% 100%; If you use  Background-size: 100%, then it should be considered using background-repeat: no-repeat; and height: auto;
Background-size: cover; also looks good to me.
You'll see the difference on smaller screens.